Makefile

```diff
@@ -28,10 +28,9 @@ endif
 ifeq ($(COVERAGE),1)
 	@go test -cover -coverprofile=$(GOPATH)/src/$@/c.out $@ -test.v
 	@go tool cover -html=$(GOPATH)/src/$@/c.out
-else
-	@echo "TESTING..."
-	@go test $@ -test.v
 endif
+	@echo "TESTING... $@"
+	@go test $@ -test.v
```

Comment:
Why this? This is going to make us run `go test` twice if we're tracking test coverage